### PR TITLE
use gltf export for all versions instead of blender process.

### PR DESCRIPTION
**Pull request by alfred316**
# Description

Change export process to use gltf exporter plugin for all UE versions. Requires marketplace download of the GLTFExporter plugin for versions before 5.1. Exported files are renamed to match up with expected format for the backend.

This completely removes the need for the blender process, so all references to that are removed.

Removed blender logo and Scene/Dynamic conversion python scripts from Resources folder.

GLTF Export also fixes various issues, such as:
- Handles skeletal mesh exports, no need for workaround
- Handles material export for scene and dynamic objects, no need for extra material work
- Has options to skip exporting things like skyboxes
- Can adjust LOD and other properties such as dimensions for textures/materials
- Faster export

Height Task ID (If applicable): T-3063, T-3849, T-1471, T-4294, T-3632

## Type of change

  - [x] Bug fix (non-breaking change which fixes an issue)
  - [x] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - [x] This change requires a documentation update

# Checklist:

  - [x] My code follows the style guidelines of this project
  - [x] I have performed a self-review of my own code
  - [x] I have commented my code, particularly in hard-to-understand areas
  - [x] I have made corresponding changes to the documentation
  - [x] My changes generate no new warnings
  - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: CognitiveVRTest/Plugins/CognitiveVR/Source/CognitiveVR/Private/CognitiveVR.cpp [hunk 1]
```diff
@@ -992,7 +992,7 @@ bool FAnalyticsProviderCognitiveVR::HasEyeTrackingSDK()
 
 bool FAnalyticsProviderCognitiveVR::TryGetRoomSize(FVector& roomsize)
 {
-#if INCLUDE_OCULUS_PLUGIN
+#ifdef INCLUDE_OCULUS_PLUGIN
```
Comment (matt-manuel) [inline CognitiveVRTest/Plugins/CognitiveVR/Source/CognitiveVR/Private/CognitiveVR.cpp:995]: Why the change from `#if` to `#ifdef`? Mostly asking out of curiosity.

**Review by matt-manuel — COMMENTED** [review]
Looks good. I won't approve since I don't know enough for that. I appreciate all the deleted lines for Blender removal though. It's a good week when you delete more lines than you write.

File: CognitiveVRTest/Plugins/CognitiveVR/Source/CognitiveVR/Private/CognitiveVR.cpp (hunk 1 above)
Comment (alfred316) [reply CognitiveVRTest/Plugins/CognitiveVR/Source/CognitiveVR/Private/CognitiveVR.cpp:995]: #ifdef checks if it has been defined, which is what we want to check. It causes an error if left as just #if

**Review by calderarchinuk — APPROVED** [review]
